package.json

```
@@ -62,7 +62,7 @@
     "bootstrap": "4.3.1",
     "core-js": "3.21.0",
     "hammerjs": "^2.0.8",
-    "iam-client-lib": "^4.2.0-alpha.3",
+    "iam-client-lib": "^4.2.1-alpha.4",
```

Could we update to canary?
```suggestion
    "iam-client-lib": "^5.0.0-alpha.7",
```